protocols/protocol/cloudstate/entity.proto

```diff
@@ -129,8 +211,19 @@ message EntitySpec {
     ServiceInfo service_info = 3;
 }
 
-// Information about the service that proxy is proxying to.
-// All of the information in here is optional. It may be useful for debug purposes.
+//*ServiceInfo* contains information about the service that the CloudState Proxy will proxy.
+//All of the information in here is optional, it is intended to be used for debug purposes.
+//
+//---
+//
+//`Specification`:
+//
+//  - A `ServiceInfo` is considered to be valid iff:
+//    - `service_name` TODO
```

Comment:
It should perhaps stated how this `service_name` differs or is similar or the same from any registered `cloudstate.Entity.service_name` where it is "The name of the service to load from the protobuf file".

If a service info can hold multiple entities, how do their `service_name`s relate to the service infos `service_name`. As it is stated that all `ServiceInfo` is optional, it might not matter at all but it might help to clarify that.